{
    "issue": {
        "title": "[Python] TestConvertStructTypes.test_from_numpy_large failing",
        "body": "***Note**: This issue was originally created as [ARROW-4181](https://issues.apache.org/jira/browse/ARROW-4181). Please see the [migration documentation](https://gist.github.com/toddfarmer/12aa88361532d21902818a6044fda4c3) for further details.*\n\n### Original Issue Description:\nThis is half of ARROW-4179 (this test should not be running in Travis CI at all). This failure appears to be a regression, and was not caught because we do not regularly run the large_memory tests\r\n\r\n`[~kszucs]` now that we have these large memory DGX machines we should run the `large_memory` unit tests at least once a day",
        "created_at": "2019-01-07T18:19:00.000Z",
        "updated_at": "2019-02-12T14:33:00.000Z",
        "labels": [
            "Migrated from Jira",
            "Component: Python",
            "Type: bug"
        ],
        "closed": true,
        "closed_at": "2019-02-12T14:32:51.000Z"
    },
    "comments": [
        {
            "created_at": "2019-02-12T14:32:51.671Z",
            "body": "***Note**: [Comment](https://issues.apache.org/jira/browse/ARROW-4181?focusedCommentId=16766079) by Uwe Korn (uwe):*\nIssue resolved by pull request 3614\n<https://github.com/apache/arrow/pull/3614>"
        }
    ]
}